python-backend/app/api/parties/party_appt/models/mine_party_appt_type.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+import re
+import uuid
+
+from sqlalchemy import func
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.orm import validates
+from sqlalchemy.schema import FetchedValue
+from app.extensions import db
+
+from ...party.models.party import Party
+from ....utils.models_mixins import AuditMixin, Base
+
+
+class MinePartyAppointmentType(AuditMixin, Base):
+    __tablename__ = "mine_party_appt_type_code"
+    mine_party_appt_type_code = db.Column(db.String(3), primary_key=True)
+    description = db.Column(db.String(100), nullable=False)
+    display_order = db.Column(db.Integer)
+    active_ind = db.Column(db.Boolean, nullable=False, server_default=FetchedValue())
+    #relationship to all appt's with a given type?
```

Comment:
Remove or make informative

Reply:
WIP placeholder for me later. will be removed later.